frontend/src/metabase/components/Popover.jsx

```diff
@@ -127,9 +127,7 @@ export default class Popover extends Component {
   }
 
   _popoverComponent() {
-    const childProps = {
-      maxHeight: this._getMaxHeight(),
-    };
+    const childProps = {};
```

Comment:
Should we also remove the `_getMaxHeight` method?

Reply:
Removed @kulyk, let's see if the changes make it through.

Thank you 🙇